src/hooks/useSearch.ts

```diff
@@ -23,3 +24,10 @@ export default function useSearch(
     }, 300);
   };
 }
+
+function setFoundProjects(projects: Project[]) {
```

Comment:
Let's create actions in store folder and move all logic with actions for entity there